docs/product-installation-guide/src/main/asciidoc/assembly_installing-on-eap-deployable.adoc

```diff
@@ -1,12 +1,12 @@
 [id='assembly_installing-on-eap-deployable']
-= ZIP file installation for {PRODUCT} on {EAP}
-
-The deployable ZIP file installation of {PRODUCT} {PRODUCT_VERSION} on {EAP} consists of two ZIP files, one for {CENTRAL} and one for {KIE_SERVER}.
+= {PRODUCT} ZIP file installation
+The {PRODUCT} ZIP file installation does not require a graphical user interface so you can install {PRODUCT} using SSH. It consists of two ZIP files, one for {CENTRAL} and one for {KIE_SERVER}.
```

Comment:
"It consists of two ZIP files..." - actually there should be also third zip file, used for standalone controller.